src/sentry/static/sentry/app/components/events/eventCause.jsx

```diff
@@ -149,42 +178,16 @@ export default createReactClass({
         return firstSeen - a[0] - (firstSeen - b[0]);
       });
     if (!commitsWithAge.length) return null;
-
-    let probablyTheCommit = commitsWithAge[0][1];
-    let commitBits = probablyTheCommit.message.split('\n');
-    let subject = commitBits[0];
-    let message =
-      commitBits.length > 1
-        ? commitBits
-            .slice(1)
-            .join('\n')
-            .replace(/^\s+|\s+$/g, '')
-        : null;
     return (
       <div className="box">
         <div className="box-header">
-          <h3>Likely Culprit</h3>
-        </div>
-        <div style={{fontSize: '0.8em', fontWeight: 'bold', marginBottom: 10}}>
-          {subject}
-        </div>
-        {!!message && (
-          <pre
-            style={{marginBottom: 10, background: 'none', padding: 0, fontSize: '0.8em'}}
-          >
-            {message}
-          </pre>
-        )}
-        <div style={{marginBottom: 20, fontSize: '0.7em', color: '#999', lineHeight: 1}}>
-          {!!probablyTheCommit.author ? (
-            <strong>{probablyTheCommit.author.name}</strong>
-          ) : (
-            <strong>
-              <em>Unknown Author</em>
-            </strong>
-          )}{' '}
-          committed <TimeSince date={probablyTheCommit.dateCreated} />
+          <h3>Suspect Commits ({commitsWithAge.length})</h3>
```

Comment:
translate

Reply:
done -- also removed a bunch of dead code that i accidentally left in